tests/v2_validation/cattlevalidationtest/core/test_network_policy.py

```diff
@@ -0,0 +1,287 @@
+from common_fixtures import *  # NOQA
+
+test_network_policy = os.environ.get(
+    'TEST_NETWORK_POLICY', "False")
+np_reason = \
+    'Rancher NetworkPolicy Manager needs to de deployed in "Default" env'
+if_network_policy = pytest.mark.skipif(test_network_policy != "True",
```

Comment:
May be you need one more reason np_reason_2="NPM tests are disabled"

And then check if the network_policy_manager stack is deployed in the env before you start running the tests and use the np_reason?